samples/tfm_integration/psa_firmware/README.rst

```diff
@@ -78,31 +97,37 @@ On MPS3 AN547:
       IMAGE1ADDRESS: 0x10080000
       IMAGE1FILE: \SOFTWARE\tfm_sign.bin ; TF-M with application binary blob
 
-4. Save the file, exit the editor, and reset the MPS3 board.
+3. Save the file, exit the editor, and reset the MPS3 board.
 
-On QEMU:
-========
+Run in QEMU:
+============
 
-Build Zephyr with a non-secure configuration (``-DBOARD=mps3_an547_ns``)
-and run it in qemu via the ``run`` command.
+.. code-block:: bash
 
-Using ``west``
+   qemu-system-arm -M mps3-an547 -device loader,file=path/to/zephyr/build/mps3_an547_ns/tfm_integration/psa_firmware/tfm_merged.hex -serial stdio  -d cpu_reset,unimp,guest_errors
 
-.. zephyr-app-commands::
-  :zephyr-app: samples/tfm_integration/psa_firmware
-  :host-os: unix
-  :board: mps3_an547_ns
-  :goals: run
-  :compact:
+or you can manually pass the ``-t run`` extra argument before `--` to the `west build` command to automatically build and run in QEMU.
```

Comment:
Why always show this way to run it, using goals: run in zephyr-app-commands, is there a reason for displaying the long command here?

The zephyr-app-commands structure is there for easier maintenance of the entire collection of readmes for the samples. I think we should use it if we can.

Reply:
I will change this based on `zephyr-app-commands`